prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -221,7 +221,29 @@ def _storage_root_uri(cls, system_id: str, storage_name: str) -> str:
           system_id: "S1"
           storage_name" "Storage"
         """
-        return cls.systems_root_uri + f"{system_id}/{storage_name}/"
+        return cls.systems_root_uri + f"{system_id}/{storage_name}"
+
+    def _find_storage_controller_data(self, storage_controllers: Dict) -> List[Dict]:
```

Comment:
Naming the parameter here `storage_controllers` is a bit misleading. When called like 

```
self._find_storage_controller_data(
                    self.redfish_obj.get(curr_storage_uri).dict
                )
```

it would contain all the data from that particular storage id since `curr_storage_uri` would be `/redfish/v1/Systems/s1/Storage/Stor1` for example. Maybe something along the lines of `storage_data` could sound more appropriate?

The if condition would also make more sense:
```
if "StorageControllers" in storage_data:
        storage_controllers_list: List[Dict] = storage_data.get("StorageControllers")
```